engine/src/object.cpp

```diff
@@ -5361,6 +5349,20 @@ void MCObject::sync_mfocus(void)
         MCmousestackptr->getcard()->mfocus(MCmousex, MCmousey);
 }
 
+bool MCObject::isancestorof(MCObject *p_object)
+{
+    if (p_object == nil)
+        return false;
```

Comment:
Should this arguably be:

```c++
MCAssert(p_object != nil);
```

Or is it expected that there will be occasions where it is useful and desirable to call this method for a null argument?